app/app_test.go

```diff
@@ -117,6 +117,9 @@ func newStartedApp(
 		GetHoneycombAPIVal:                   "http://api.honeycomb.io",
 		GetInMemoryCollectorCacheCapacityVal: config.InMemoryCollectorCacheCapacity{CacheCapacity: 10000},
 		AddHostMetadataToTrace:               enableHostMetadata,
+		TraceIdFieldNames:								 		[]string{"trace.trace_id"},
+		SpanIdFieldNames:								 			[]string{"trace.span_id"},
```

Comment:
Since refinery doesn't currently use the span_id name, let's take it out of this PR. If we do something later, we can add it then with no harm.